apps/dashboard/src/components/workflow-row.tsx

```diff
@@ -95,7 +95,7 @@ export const WorkflowRow = ({ workflow }: WorkflowRowProps) => {
             </TableCell>
           </TooltipTrigger>
           <TooltipPortal>
-            <TooltipContent>{new Date(workflow.updatedAt).toUTCString()}</TooltipContent>
```

Comment:
apologies webstorm did not push this change commit to [my last PR](https://github.com/novuhq/novu/pull/6948) 😅